### PR TITLE
Update thankyou.js

### PR DESCRIPTION
**Pull request by RahulMetre03**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://rahulmetre03.github.io/gci18.fossasia.org/)

- Fixes #486 

## Description / Changes
ADDED AND ENHANCED 'THANK YOU FOR CONTRIBUTING' IN KANNADA

### After:

- - - - - - - - - - - -

### CODE REVIEW

File: js/thankyou.js [hunk 1]
```diff
@@ -28,6 +28,7 @@
 'Terimakasih atas kontribusi anda!',
 'Köszönjük a hozzájárulást!',
 'Takk fyrir þitt framlag!'
```
Comment (Ryan10145) [inline js/thankyou.js]: There should be a comma between elements in an array.

Comment (RahulMetre03) [reply js/thankyou.js]: It's done

Comment (akshatnitd) [inline js/thankyou.js]: Add a comma a the end of this line

Comment (RahulMetre03) [reply js/thankyou.js]: Added a comma

File: js/thankyou.js [hunk 2]
```diff
@@ -27,8 +27,11 @@
 'Go raibh maith agat as do chuid ranníocaíochtaí!',
 'Terimakasih atas kontribusi anda!',
 'Köszönjük a hozzájárulást!',
+'Takk fyrir þitt framlag!',
```
Comment (abishekvashok) [inline js/thankyou.js]: Appearing two times. Please delete this line

**Comment by abishekvashok** [comment]
@SIMRAN1 most students are new to opensource, it's alright when they start that they might mess up a few things such as commit messages, we could do that for them, eg) the squash and merge option provides a way to alter the commit message.

We shouldn't be too strict.

File: js/thankyou.js (hunk 2 above)
Comment (RahulMetre03) [reply js/thankyou.js]: Okay

**Comment by RahulMetre03** [comment]
@abishekvashok Its done!:)

**Comment by SIMRAN1** [comment]
okk @abishekvashok !

**Pull request merged**